File: My Leet Code Python Solutions/Solution1523.py
#1523. Count Odd Numbers in an Interval Range
class Solution:
    def countOdds(self, low: int, high: int) -> int:
        total = high - low + 1
        if(low % 2 == 1 and high % 2 == 1):
            return int(total/2+1)
        return int(total/2)

File: My Leet Code Java Solutions/Solution1523.java
//1523. Count Odd Numbers in an Interval Range

class Solution1523 {
    public int countOdds(int low, int high) {
        int total = high - low + 1;
        if (low % 2 == 1 && high % 2 == 1) {
            return total / 2 + 1;
        } else
            return total / 2;
    }
}